tests/integration/cattletest/core/common_fixtures.py

```diff
@@ -33,13 +37,42 @@ def _client_for_user(name, accounts):
                            secret_key=accounts[name][1])
 
 
+def client_for_project(project):
+    access_key = random_str()
+    secret_key = random_str()
+    admin_client = _admin_client()
+    active_cred = None
+    account = project
+    for cred in account.credentials():
```

Comment:
You are right in that it will be random for both access and secrete key. However If there were ones with the same ```random_str()``` they would get reused. This was code used from creating a user. I could remove the for loop and the result should be the same. Assuming that ```random_str()``` is always random.